ohtu2021-kvantti/WebMark
# Check if task exists

## Summary
Implements task #105. Exception handling is implemented to worker so that version-molecule pairs does not stay in queue permanently if analysis fails to exception.

## How to test
Need of makemigrations & migrate because one field is added to database. Check that correct popup message is shown when sending algorithm version to analyze queue. After sending task, user should land in same page (correct algorithm details view, with correct version).
